docs/content/tutorials/quickstart.md

```diff
@@ -0,0 +1,242 @@
+---
+layout: doc_page
+---
+
+# Druid Quickstart
+
+In this quickstart, we will download Druid, set up it up on a single machine, load some data, and query the data.
+
+## Prerequisites
+
+You will need:
+
+  * Java 7 or higher
+  * Linux, Mac OS X, or other Unix-like OS (Windows is not supported)
+  * 8G of RAM
+  * 2 vCPUs
+
+On Mac OS X, you can use [Oracle's JDK 
+8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) to install 
+Java.
+
+On Linux, your OS package manager should be able to help for Java. If your Ubuntu-
+based OS does not have a recent enough version of Java, WebUpd8 offers [packages for those 
+OSes](http://www.webupd8.org/2012/09/install-oracle-java-8-in-ubuntu-via-ppa.html).
+
+## Getting started
+
+To install Druid, issue the following commands in your terminal: 
+
+```bash
+curl -O http://static.druid.io/artifacts/releases/druid-0.9.0-bin.tar.gz
+tar -xzf druid-0.9.0-bin.tar.gz
+cd druid-0.9.0
+```
+
+In the package, you should find:
+
+* `LICENSE` - the license files.
+* `bin/` - scripts that will be used for this setup.
+* `conf/*` - template configurations for a clustered setup.
+* `conf-quickstart/*` - configurations for this quickstart.
+* `extensions/*` - all Druid extensions.
+* `hadoop-dependencies/*` - Druid Hadoop dependencies.
+* `lib/*` - all included software packages for core Druid.
+* `quickstart/*` - files useful for this quickstart.
+
+## Start up Zookeeper
+
+Druid currently has a dependency on [Apache ZooKeeper](http://zookeeper.apache.org/) for distributed coordination. You'll 
+need to download and run Zookeeper.
+
+```bash
+curl http://www.gtlib.gatech.edu/pub/apache/zookeeper/zookeeper-3.4.6/zookeeper-3.4.6.tar.gz -o $zookeeper-3.4.6.tar.gz
+tar xzf $zookeeper-3.4.6.tar.gz
```

Comment:
I don't see why there's a $ in front of zookeeper-3.4.6.tar.gz on this line and the one before.